src/elements/content-sidebar/SidebarContentInsights.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+
+import ContentInsightsSummary from '../../features/content-insights/ContentInsightsSummary';
+import { GraphData } from '../../features/content-insights/types';
+// @ts-ignore Module is written in Flow
+import messages from '../common/messages';
+// @ts-ignore Module is written in Flow
+import SidebarSection from './SidebarSection';
+// @ts-ignore Module is written in Flow
+// eslint-disable-next-line import/no-named-as-default, import/no-named-as-default-member
```

Comment:
We should ideally try to avoid ever disabling our linting rules. It looks like we have other components that import this module without issue. What was the reasoning here?

Reply:
The issue appears to be with the tooling, eslint seems to choke on the `type` import in `withErrorHandling` and it seems like I can work around it by moving the file to a `hocs` folder and providing an `index.js` but refactoring  that code just to solve this typescript/flow import issue seems like overkill. I'll try to add a comment to describe why these linting rules are disabled